braindecode/datautil/xy.py

```diff
@@ -69,7 +69,7 @@ def create_from_X_y(
     windows_datasets = create_fixed_length_windows(
         base_datasets,
         start_offset_samples=0,
-        stop_offset_samples=0,
+        stop_offset_samples=None,
```

Comment:
API change?

Reply:
That's correct. Is there a recommended way to flag API changes? I added a warning here: https://github.com/braindecode/braindecode/pull/152/files#diff-c031004527cfc6287fbb1741b7f3e1ddR231